Gemfile

```diff
@@ -29,7 +29,7 @@ gem 'mechanize'
 gem 'sass'
 gem 'jquery-rails'
 
-gem 'industry_mapping_client', git: 'git://github.com/GovWizely/industry_mapping_client.git', tag: 'v0.1.1'
+gem 'industry_mapping_client', git: 'git://github.com/GovWizely/industry_mapping_client.git'
```

Comment:
@calumhalcrow Just an oversight, but when adding Gemfile.lock I found my recently upgraded version of bundler appended the following version information to the file:

```+
BUNDLED WITH
   1.10.6```

As a workaround, I removed the lines, downgraded bundler to version 1.5.3 -- the version we're using on OpsWorks (the aws default), tested, and committed Gemfile.lock. 

I found some heated discussion of the 'BUNDLED WITH ...' issue online. I'll ping you about it later.